README.md

```diff
@@ -32,10 +32,10 @@ Work within the WordPress admin with an interface you already know.
 Add from a growing list of available fields to your custom blocks.
 
 ### Simple Templating ###
-Let the plugin do the heavy lifting so you can use familiar WordPress development practices to build block templates.
+Let the plugin do the heavy lifting so you can use the built-in editor, or familiar WordPress development practices to build block templates.
```

Comment:
The README doesn't really mention templating much. It mainly links to our [documentation](https://developer.wpengine.com/genesis-custom-blocks/).

So I changed this to mention the built-in editor first.

Reply:
But this could use a screenshot of the built-in editor, and it should be before the screenshot of the PHP template.